includes/class-boldgrid-inspirations-purchase-for-publish.php

```diff
@@ -1417,14 +1465,18 @@ public function asset_needs_purchase( $asset, $asset_type, $args = array() ) {
 
 				// SELECT post_title where post_content like
 				// '%2015/02/google-maps-int-1410976385-pi.jpg%'
-				$query = $wpdb->prepare("
-					SELECT	`ID`
-					FROM	$wpdb->posts
-					WHERE	`post_content` LIKE %s AND
-							`post_status` IN $post_status AND
-							`post_type` IN ('page','post')
-					",
-					'%' . $wpdb->esc_like( $file_name_to_query ) . '%'
+				$placeholders = $this->get_prepare_placeholders(
+					array(
+						$post_status,
+						'%' . $wpdb->esc_like( $file_name_to_query ) . '%',
+					)
+				);
+
+				$query = $wpdb->prepare(
+					"SELECT	`ID` FROM $wpdb->posts WHERE `post_content` LIKE %s AND `post_status` IN ( " .
```

Comment:
I'm trying to peer review and figure out what has changed... The PR can be much cleaner and easier to understand by not reformatting, unless it needs to be.